src/utils/decodeEvents.js

```diff
@@ -1,22 +1,42 @@
 import BRIDGE_ABI_V2 from '@/constants/abis/bridge.json'
 import BRIDGE_ABI_V1 from '@/constants/abis/bridge_v1.json'
 import BRIDGE_ABI_V0 from '@/constants/abis/bridge_v0.json'
+import NFT_BRIDGE from '@/constants/abis/nft-bridge.json'
+import { TOKEN_TYPE_ERC_20, TOKEN_TYPE_ERC_721 } from '@/constants/tokenType'
 
-export function decodeCrossEvent(web3, receipt) {
+export function decodeCrossEvent(web3, receipt, tokenType) {
+  switch (tokenType) {
+    case TOKEN_TYPE_ERC_20:
+      return decodeERC20CrossEvent(web3, receipt)
+    case TOKEN_TYPE_ERC_721:
+      return decodeERC721CrossEvent(web3, receipt)
+    default:
```

Comment:
same here, we should maybe throw an error, right? since it'd be an unsupported token type (it won't necessarily behave like ERC20)